### PR TITLE
fix snyk failue in high level

### PR DESCRIPTION
**Pull request by Ximinhan**
fix hight level warning
test run for publish-rpms: https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/hack/job/ximhan/job/build%252Fpublish-rpms/4/

### CODE REVIEW

**Review by joepvd — COMMENTED** [review]
Nice! One comment. 

And did you manage to test this? Or what is your deployment plan to mitigate any issues?

**Comment by Ximinhan** [comment]
the medium error left you can see is https://github.com/openshift-eng/aos-cd-jobs/actions/runs/4752228086/jobs/8442322938?pr=3642 related to Path Traversal  I didn't find a way to fix it, any input variable pass in os.path it will think vulnerability

File: hacks/plashet/build-plashet.py [hunk 1]
```diff
@@ -315,7 +315,7 @@ async def cmd_assert_async(cmd: Union[Sequence[str], str], check: bool = True, *
     else:
         cmd_list = cmd
     LOGGER.info("Executing:cmd_assert_async %s", cmd_list)
-    proc = await asyncio.subprocess.create_subprocess_exec(cmd_list[0], *cmd_list[1:], **kwargs)
+    proc = await asyncio.subprocess.create_subprocess_exec(shlex.quote(cmd_list[0]), shlex.quote(cmd_list[1:]), **kwargs)
```
Comment (vfreex) [inline hacks/plashet/build-plashet.py:318]: using shlex.quote in asyncio.subprocess.create_subprocess_exec seems wrong to me. shlex.quote is used for escaping special characters for shell, however asyncio.subprocess.create_subprocess_exe doesn’t invoke the command in a shell at all.

File: pyartcd/pyartcd/pipelines/prepare_release.py [hunk 2]
```diff
@@ -618,11 +618,11 @@ def fields_transform(fields):
     def _render_jira_template(fields: Dict, template_vars: Dict):
         fields.copy()
         try:
-            fields["summary"] = jinja2.Template(fields["summary"]).render(template_vars)
+            fields["summary"] = jinja2.Template(fields["summary"], autoescape=True).render(template_vars)
```
Comment (vfreex) [inline pyartcd/pyartcd/pipelines/prepare_release.py:621]: This warning doesn't make sense to me either. `autoescape` is for escaping HTML. I don't think we can use HTML in JIRA.

File: hacks/plashet/build-plashet.py (hunk 1 above)
Comment (Ximinhan) [reply hacks/plashet/build-plashet.py]: snyk think the first command of asyncio.subprocess.create_subprocess_exe is a shell cmd and has a vulnerability, it not smart to check the env.

File: pyartcd/pyartcd/pipelines/prepare_release.py (hunk 2 above)
Comment (Ximinhan) [reply pyartcd/pyartcd/pipelines/prepare_release.py:621]: Similarly, snyk thinks the jinja2.Template method should have autoescape argument, it didn't check the context

**Review by vfreex — APPROVED** [review]
/lgtm

**Pull request merged**